tests/fv3_conf/control_run.IN

```diff
@@ -88,8 +88,10 @@ fi
 if [ $TILEDFIX = .true. ]; then
     if [ $CPLLND == .true. ]; then
         cp    @[INPUTDATA_ROOT]/FV3_fix_tiled/@[ATMRES]/@[ATMRES]*.nc INPUT/.
+        cp    /scratch2/NCEPDEV/stmp1/Sanath.Kumar/repo_viirs30s_bnuv2/sfc.@[ATMRES]/*color* INPUT/. 
```

Comment:
This line needs to be updated to use the input data directory shown in line 90.

Reply:
This line needs to be removed when the soil color data are put in @[INPUTDATA_ROOT]/FV3_fix_tiled